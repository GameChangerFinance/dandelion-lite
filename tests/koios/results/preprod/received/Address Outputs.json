{
  "code": "42P01",
  "details": null,
  "hint": null,
  "message": "relation \"public.address\" does not exist"
}
