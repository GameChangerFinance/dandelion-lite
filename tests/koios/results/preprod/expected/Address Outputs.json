[
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 1,
    "value": "80000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 3,
    "value": "40000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 4,
    "value": "40000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 5,
    "value": "5000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 6,
    "value": "5000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": true,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 2,
    "value": "80000000"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3278627,
    "block_time": 1741974426,
    "epoch_no": 203,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "6d0a5544568b37d9c7d3fb08fb6efedd44e18840f36ace1c6c955a087c831477",
    "tx_index": 0,
    "value": "1728310"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3335165,
    "block_time": 1743457217,
    "epoch_no": 206,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "1fc72f03f349d1e19d140dcef4cfece2c0bc89a04c7afcaa26d1a07c7e0c7e31",
    "tx_index": 8,
    "value": "6649715"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3335165,
    "block_time": 1743457217,
    "epoch_no": 206,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "1fc72f03f349d1e19d140dcef4cfece2c0bc89a04c7afcaa26d1a07c7e0c7e31",
    "tx_index": 7,
    "value": "32998548"
  },
  {
    "address": "addr_test1qzae9j9kvdehj896c0nruwm9h98jmyaqwjesc4r6pr604nadqwj2u3djrag0mene2cm9elu5mdqmcz9zc2rzgq7c5g6q96auyp",
    "block_height": 3335165,
    "block_time": 1743457217,
    "epoch_no": 206,
    "is_spent": false,
    "payment_cred": "bb92c8b66373791cbac3e63e3b65b94f2d93a074b30c547a08f4facf",
    "stake_address": "stake_test1uzks8f9wgkep758aueu4vdjul72dksdupz3v9p3yq0v2ydqx8me8y",
    "tx_hash": "1fc72f03f349d1e19d140dcef4cfece2c0bc89a04c7afcaa26d1a07c7e0c7e31",
    "tx_index": 1,
    "value": "2146460"
  }
]
